app/build.gradle

```diff
@@ -85,18 +88,21 @@ jacocoAndroidUnitTestReport {
 }
 
 buildscript {
-    ext.support_library_version = '27.1.0'
+    ext.support_library_version = '27.1.1'
     ext.dagger_version = '2.13'
     ext.wolmo_version = '2.0.1'
-    ext.butterknife_version = '9.0.0-SNAPSHOT'
```

Comment:
Was this number lowered for any particular reasons?

Reply:
It does not exist anymore :thinking: